dali/pipeline/operators/argument.h

```diff
@@ -56,13 +52,9 @@ class ValueInst : public Value {
     this->type_ = TypeTable::GetTypeID<T>();
   }
 
-  std::string ToString() const override {
-    return to_string(val_);
-  }
+  std::string ToString() const override { return to_string(val_); }
```

Comment:
I thought you didn't like one liners like that :)

Reply:
Yup, auto-format did. Should probably unwrap it :P

Reply:
ping

Reply:
Done